apps/yapms/src/lib/stores/Candidates.ts

```diff
@@ -33,6 +33,8 @@ export const CandidatesStore = writable<Candidate[]>([
 	}
 ]);
 
+export const DefaultCountsStore = writable<Map<string, number>>(new Map<string, number>());
```

Comment:
The DefaultCountsStore is redundant data, as you've already done all the work to collect this information into the CandidateStore.